src/js/azdata/azdataQueryPlan.js

```diff
@@ -296,6 +296,26 @@ azdataQueryPlan.prototype.init = function (queryPlanConfiguration) {
     };
     this.keyHandler.bindKey(arrowDownKey, selectBottom);
 
+    const enterKey = 13;
```

Comment:
Both enter and space key should collapse or expand the node based on the previous state.

Example: If I press enter/space on an expanded node, it should collapse and vice-versa.